_posts/2017-08-25-introducing-sandboxfs.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: posts
+title: Introducing sandboxfs
+---
+
+sandboxfs is a new project to improve the way sandboxing works in Bazel by
```

Comment:
Small nit - let's use bold for "sandboxfs" on this first line. It'll make the lowercase at the start of the sentence look less awkward.

Reply:
Done.